src/TransparentVerifiableProxy.sol

```diff
@@ -46,7 +53,7 @@ contract TransparentVerifiableProxy is Proxy, Initializable {
         payable
         initializer
     {
-        require(msg.sender == creator, "Unauthorized initialization");
+        require(msg.sender == verifiableProxyCreator, "Unauthorized initialization");
```

Comment:
As discussed in our 1:1, because construction and initialization are atomic, we can skip this check, as long as we document this expectation.